.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
```

Comment:
Maybe add MessageBird to name, node.js package sounds a bit generic

Reply:
✅